Gems/Atom/Feature/Common/Code/Source/CoreLights/PointLightFeatureProcessor.cpp

```diff
@@ -120,6 +120,22 @@ namespace AZ
             {
                 m_lightData.GetData<0>(handle.GetIndex()) = m_lightData.GetData<0>(sourceLightHandle.GetIndex());
                 m_lightData.GetData<1>(handle.GetIndex()) = m_lightData.GetData<1>(sourceLightHandle.GetIndex());
+
+                auto& light = m_lightData.GetData<0>(handle.GetIndex());
+                for (int i = 0; i < PointLightData::NumShadowFaces; ++i)
+                {
+                    ShadowId shadowId = ShadowId(light.m_shadowIndices[i]);
+                    if (shadowId.IsValid())
+                    {
+                        // Since the source light has a valid shadow, a new shadow must be generated for the cloned light.
+                        ProjectedShadowFeatureProcessorInterface::ProjectedShadowDescriptor originalDesc =
+                            m_shadowFeatureProcessor->GetShadowProperties(shadowId);
+                        ShadowId cloneShadow = m_shadowFeatureProcessor->AcquireShadow();
+                        light.m_shadowIndices[i] = cloneShadow.GetIndex();
+                        m_shadowFeatureProcessor->SetShadowProperties(cloneShadow, originalDesc);
+                    }
+                }
+
```

Comment:
Just curious, did we have issue with shadow for point light before without this code?